corona-ide-core/src/main/java/com/coronaide/core/services/impl/DatastoreService.java

```diff
@@ -333,7 +329,7 @@ private void storeVersion(Path moduleDirectory, Version version, String datastor
         allVersions.add(datastoreKey, newVersion);
 
         // Write out to file
-        try (JsonWriter writer = gson.newJsonWriter((new FileWriter(versionFile, false)))) {
+        try (JsonWriter writer = gson.newJsonWriter(new FileWriter(versionFile, false))) {
```

Comment:
Haha, yeah it was complaining there was an extra set, and if it was asking to add I would have just told it to go away with the false positive button :P